mu-plugins/blocks/navigation/index.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Update the Navigation block to work with a `menuSlug`
```

Comment:
It wasn't exactly clear to me how this block was supposed to work since it's in the block folder but is really a _filter_ on top of the navigation block.

Can we update this description to explain that better?

Reply:
[Done](https://github.com/WordPress/wporg-mu-plugins/pull/443/commits/cba28f4bb5a76d7138d36f59e9ffbdfc97be71bb), let me know if it makes sense :)